docusaurus/docs/Android/03-ui/02-general-customization/02-chatui.mdx

```diff
@@ -110,15 +159,31 @@ The methods mentioned above are suspending functions. If you don't use coroutine
 
 **If you don't want to use avatars from the internet and only use the default avatar**, you can call `createDefaultUserBitmap` inside `createUserBitmap` in your custom implementation of `AvatarBitmapFactory`:
 
-```
-public open suspend fun createUserBitmap(
-        user: User,
-        style: AvatarStyle,
-        @Px avatarSize: Int,
-    ): Bitmap? {
+<Tabs>
+<TabItem value="kotlin" label="Kotlin">
+
+```kotlin
+object : AvatarBitmapFactory(context) {
+    override suspend fun createUserBitmap(user: User, style: AvatarStyle, avatarSize: Int): Bitmap? {
         return createDefaultUserBitmap(user, style, avatarSize)
     }
+}
+```
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+new AvatarBitmapFactory(context) {
+    @Nullable
+    @Override
+    public Object createUserBitmap(@NonNull User user, @NonNull AvatarStyle style, int avatarSize, @NonNull Continuation<? super Bitmap> $completion) {
```

Comment:
```suggestion
    public Bitmap createUserBitmap(@NonNull User user, @NonNull AvatarStyle style, int avatarSize, @NonNull Continuation<? super Bitmap> $completion) {
```

Reply:
for some reason, it doesn't allow changing the return type to Bitamp. Leaving Object for now